### DISCUSSION TITLE
curl: option --no-progress-meter: is unknown

### DISCUSSION THREAD

**Discussion by lambertbeekhuis** [tailpress/tailpress#164] [Q&A]
When I run tailpress, and want to add wordpress as well, I get the message:

curl: option --no-progress-meter: is unknown
curl: try 'curl --help' or 'curl --manual' for more information
mkdir: /Users/lambert/Sites/urentellertailpress/wp-content/themes: No such file or directory

I installed curl using brew, and got version:
curl 7.64.1 (x86_64-apple-darwin20.0) libcurl/7.64.1 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.41.0

How to continue?

Thanks in advance, Lambert

**Comment by jeffreyvr** [comment]
The --no-progress-meter is available with curl since version 7.67.0. Are you perhaps using an older version?

Edit: I now see you have since updated your post with the curl vesion. Solution would be to update curl.
